### ISSUE TITLE
Crash, possibly related to DoubleSlabs mod

### ISSUE THREAD

**Issue by NiallDoherty** [ldtteam/minecolonies#6133]
### Prerequisites 

- [x ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x ] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [ x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.3
- MineColonies Version: minecolonies:0.13.458-ALPHA
- Structurize Version: 0.13.96-ALPHA

## Problem description

The server crashes

### Notes or related things

Someone on the Valhelsia modpack discord mentioned not to use vertical slabs to fix MineColonies crashing, and it mentions the DoubleSlab mod in the crash report so there must be some incompatibility there.

---
## Viewers

* Add a thumbs-up to the issue to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Were you able to reproduce this? Like Minecolonies + Double Slabs mod = crash?

**Comment by NiallDoherty** [comment]
Not really, just anecdotal evidence. The servers been up a few days and this is the first time it's crashed so there's more to it than just having the two mods installed.

**Comment by NiallDoherty** [comment]
Deleted DoubleSlabs and it appears to be working fine now

**Comment by Raycoms** [comment]
Is there any way to reproduce this?

**Comment by Raycoms** [comment]
Easily?
